lib/src/flutter_uploader.dart

```diff
@@ -71,20 +71,16 @@ class FlutterUploader {
   /// In order to clear the list, you can use [clearUploads] following by a
   /// re-subscription to this stream.
   Stream<UploadTaskResponse> get result {
-    return _resultStream ??= _resultChannel
-        .receiveBroadcastStream()
-        .map<Map<String, dynamic>>((event) => Map<String, dynamic>.from(event))
-        .map(_parseResult);
+    return _resultStream ??=
+        _resultChannel.receiveBroadcastStream().map<Map<String, dynamic>>((event) => Map<String, dynamic>.from(event)).map(_parseResult);
```

Comment:
I need to revert this change again. @ened is the line length 80?

Reply:
Probably is. You can run `flutter format` to get the codebase into the accepted format.